rdflib/term.py

```diff
@@ -1383,6 +1383,14 @@ def _unhexlify(value):
         value = value.encode()
     return unhexlify(value)
 
+def _parseBoolean(value):
+    new_value = value.lower()
+    if new_value == 'true' or new_value=='1':
```

Comment:
`new_value=='1'` -> `new_value == '1'`

Maybe you just use `new_value in ['true', '1']` resp. `new_value not in ['false', '0']`.

Reply:
Added these changes in the new commit 👍